Add support for configuring Ingress class (#1851)
strimzi/strimzi-kafka-operator: documentation/book/con-kafka-broker-external-listeners-ingress.adoc

For more information on using `Ingress` to access Kafka, see xref:proc-accessing-kafka-using-ingress-{context}[].

> = Configuring `Ingress` class

Reviewer: ```suggestion
= Configuring the `Ingress` class
```

> By default, the `Ingress` class will be set to `nginx`.

Reviewer: ```suggestion
By default, the `Ingress` class is set to `nginx`.
```
